libcxxabi/src/fallback_malloc.cpp

```diff
@@ -142,7 +142,7 @@ void* fallback_malloc(size_t len) {
 
     // Check the invariant that all heap_nodes pointers 'p' are aligned
     // so that 'p + 1' has an alignment of at least RequiredAlignment
-    assert(reinterpret_cast<size_t>(p + 1) % RequiredAlignment == 0);
+    _LIBCPP_ASSERT_UNCATEGORIZED(reinterpret_cast<size_t>(p + 1) % RequiredAlignment == 0, "");
```

Comment:
Why are we not using `DEMANGLE_ASSERT` in this file?

Reply:
Because that part of libc++abi is not part of the demangler. The demangler is a special part of libc++abi that is shared with the rest of LLVM and that’s why we introduced a macro there.